pnpm-lock.yaml

```diff
@@ -6372,9 +6363,11 @@ snapshots:
   '@babel/plugin-transform-modules-commonjs@7.24.6(@babel/core@7.26.0)':
     dependencies:
       '@babel/core': 7.26.0
-      '@babel/helper-module-transforms': 7.24.6(@babel/core@7.26.0)
+      '@babel/helper-module-transforms': 7.26.0(@babel/core@7.26.0)
```

Comment:
I don't like that pnpm automatically updates the latest sub-dependencies each time -- we may get bugs without really knowing.

I don't think there's anything we can do about this...